src/components/organisms/Header/Header.tsx

```diff
@@ -1,21 +1,33 @@
 import React from 'react';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import { useHistory } from 'react-router-dom';
-import { Bubble, Avatar } from '../../../components';
+import { Avatar } from '../../../components';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faHome } from '@fortawesome/free-solid-svg-icons';
+import { TabNavigation } from '../../../components';
 
-const HeaderWrapper = styled.div`
-  width: 100vw;
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  background-size: cover;
-  background: ${({ theme }) => theme.main.colors.purpleDark};
-  margin-top: -50px;
-  .icon {
-    color: ${({ theme }) => theme.main.colors.purple};
-  }
+const HeaderWrapper = styled.div(({ theme }) => {
+  const { background } = theme;
+
+  return css`
+    width: 100vw;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    background-size: cover;
+    background-color: ${background.primary};
+    margin-top: -50px;
+    .icon {
+      background-color: ${background.primary};
```

Comment:
What's the point of setting the background a second time for a child element? Should work the same if this is taken out

Reply:
Good point. I think that was leftover Frome when I used the bubbles